<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="MainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonEG">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>EG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOG">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>OG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUG">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>UG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGarage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Garage</string>
   </property>
  </widget>
  <widget class="QLabel" name="temperature">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="humidity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainWindow</class>
   <extends>QWidget</extends>
   <header>mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
